Aquality.Appium.Mobile.Template/Screens/WebLogin/AndroidLoginScreen.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Aquality.Appium.Mobile.Applications;
+using Aquality.Appium.Mobile.Screens.ScreenFactory;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Appium;
+
+namespace Aquality.Appium.Mobile.Template.Screens.WebLogin
+{
+    [ScreenType(PlatformName.Android)]
+    public sealed class AndroidLoginScreen : LoginScreen
+    {
+        public AndroidLoginScreen() : base(By.XPath("//android.widget.Button[@text='Log in']"))
+        {
+        }
+
+        protected override By UsernameTxbLoc => MobileBy.XPath("//android.widget.EditText[@resource-id=\"user_email\"]");
```

Comment:
You should use `'` in the locators. It is the same as `"`, but you don't need to escape it with `\`